schema/deploy/trigger_functions/commit_form_change.sql

```diff
@@ -20,7 +20,7 @@ begin
   end if;
 
   schema_table := quote_ident(new.form_data_schema_name) || '.' || quote_ident(new.form_data_table_name);
-  keys := (select array_to_string(array(select quote_ident(key) from jsonb_each(new.new_form_data)), ','));
+  keys := (select array_to_string(array(select quote_ident(lower(regexp_replace(key, '([A-Z])','_\1', 'g'))) from jsonb_each(new.new_form_data)), ','));
```

Comment:
should we refactor this in a small function on the `cif_private` schema? it would be more readable in this file